Package.swift

```diff
@@ -7,15 +7,15 @@ let package = Package(
     name: "Loggie",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v10)
+        .iOS(.v11)
     ],
     products: [
         .library(
             name: "Loggie",
             targets: ["Loggie"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.6.4"))
```

Comment:
How come you changed this as well?

Reply:
The previous package dependency version was `5.2.0`, which was increased in the latest two releases.
Still, this library works fine even on earlier versions of `Alamofire` (e.g., `5.2.0` and up).  

With this change, I want to avoid forcing developers to update the Alamofire to the latest version - if that is not necessary, even though it is recommended.

@goranbrl, we can discuss this in more detail, but if you agree with this based on the provided argument, we can merge this change.